pronto_mia/lib/ui/views/startup/startup_viewmodel.dart

```diff
@@ -44,4 +49,14 @@ class StartUpViewModel extends BaseViewModel {
       _navigationService.replaceWith(Routes.loginView);
     }
   }
+
+  Future<void> _handlePushNotifcations(bool isAuthenticated) async {
+    final notificationsAuthorized =

```

Comment:
I think it is better this way than after the authentication. Since this way the popup will be created on page load.